tests/session.py

```diff
@@ -3,44 +3,68 @@
 from sqlalchemy import create_engine
 
 from clickhouse_sqlalchemy import make_session
-from tests.config import uri, native_uri, system_native_uri
-
-session = make_session(create_engine(uri))
+from tests.config import (
+    http_uri, native_uri, system_native_uri, system_http_uri,
+)
 
+http_session = make_session(create_engine(http_uri))
+http_stream_session = make_session(create_engine(http_uri + '?stream=1'))
```

Comment:
It's better to create stream session in `StreamingHttpTestCase`. This session is not shared between another test cases.

Reply:
> This session is not shared between another test cases.

It should be.

Best-case would be to parametrize all the relevant tests (the ones doing any querying) on all the sessions; right now this is a bit hard to do; and there are other sessions that need to be added (e.g. parametrization by the serialization format).

Any suggestions on how to better parametrize tests in this framework?

Reply:
OK, I'll think about it.